src/loaders/google_map_loader.js

```diff
@@ -58,15 +58,33 @@ export default (bootstrapURLKeys, heatmapLibrary) => {
       }
     }
 
+    // Support for older version using heatMapLibrary option
+    if (heatmapLibrary) {
+      bootstrapURLKeys.libraries
+        ? bootstrapURLKeys.libraries.append('visualization')
```

Comment:
Here you're appending even if the library is present!

Reply:
because duplicates get cleaned out anyways in:
```javscript
    if (bootstrapURLKeys.libraries) {
      bootstrapURLKeys.libraries = bootstrapURLKeys.libraries.filter(
        (lib, i) =>
          bootstrapURLKeys.libraries.indexOf(lib) === i &&
          googleMapsLibs.includes(lib)
      );
    }
```